serve.py

```diff
@@ -1,34 +1,110 @@
 import asyncio
-
+import os
 from sanic import Sanic
-from sanic.log import logger, error_logger, access_logger
+from sanic.log import logger
 from sanic import response
-from sanic_cors import CORS, cross_origin
+from sanic_cors import CORS
 import requests
-from sanic.exceptions import abort
 
-app = Sanic("redirector")
+production = True and 'DEV628cc89a6444' not in os.environ
```

Comment:
```suggestion
production = 'DEV628cc89a6444' not in os.environ
```
I guess left from some debugging